Userland/Libraries/LibWeb/Layout/SVGSVGBox.cpp

```diff
@@ -27,14 +27,13 @@ void SVGSVGBox::prepare_for_replaced_layout()
     if (dom_node().has_attribute(HTML::AttributeNames::width) && dom_node().has_attribute(HTML::AttributeNames::height)) {
         Optional<CSSPixels> w;
         Optional<CSSPixels> h;
-        if (auto width = HTML::parse_dimension_value(dom_node().attribute(HTML::AttributeNames::width))) {
-            if (width->has_length())
-                w = width->to_length().to_px(*this);
-        }
-        if (auto height = HTML::parse_dimension_value(dom_node().attribute(HTML::AttributeNames::height))) {
-            if (height->has_length())
-                h = height->to_length().to_px(*this);
-        }
+        auto width = parse_css_value(CSS::Parser::ParsingContext { document() }, dom_node().attribute(Web::HTML::AttributeNames::width), CSS::PropertyID::Width);
```

Comment:
We can get away with just creating the ParsingContext once and passing it to both `parse_css_value()` calls.